frontend-client/src/components/BodyMachinesTeamCyanGenerator.vue

```diff
@@ -0,0 +1,165 @@
+        <div class="mb-1 col-xl-12 col-md-3"
+             :class="[machine.mtype+'-station']"
+             v-for="(machine,index) in machinesCyan" 
+             :key="index" 
+        >
+          <div class="d-flex align-content-center ml-3" >
+            <div class="base-image-container mr-1">
+                    <figure>
+                      <div class="station-specific-info-container text-center"
+                           :class="setMachineClass(machine)" 
+                      >
+                        <span class="text-secondary text-center">{{machine.name}}</span>
+                      </div>
+                      <figcaption class="text-center machine-zone">{{machine.zone}}
+                        <br>
+                        <span v-if="currPhase === 'Setup'">{{machine.rotation}}°</span>
+                      </figcaption>
+                    </figure>
+            </div>
+            <div class="machine-info d-flex flex-column">
+                  <span :class="setStateClass(machine.state)">
+                    {{machine.state}}
+                  </span>
+                  <div v-if="machine.mtype === 'RS'" class="d-flex my-1">
+                    <span>{{machine['bases-added'] - machine['bases-used']}}</span>
+                    <div class="ml-2 d-flex">
+                      <div 
+                        class="rs-color-container mr-1"
+                        :class="setRsColor(machine['rs-ring-colors'][0])">
+                        <span class="invisible">color</span>
+                      </div>
+                      <span>{{showPreparedColorInfo(machine['rs-ring-colors'][0])}}</span>
+                      <div 
+                        class="rs-color-container mx-1"
+                        :class="setRsColor(machine['rs-ring-colors'][1])">
+                        <span class="invisible">color</span>
+                      </div>
+                      <span>{{showPreparedColorInfo(machine['rs-ring-colors'][1])}}</span>
+                    </div>
+                  </div>
+                  <span>{{machine.name}}</span>
+            </div>
+          </div>
+        </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapState } from 'vuex';
+export default {
+  name: 'BodyMachinesTeamCyanGenerator',
+  computed: {
+    ...mapState({
+      machinesCyan: state => state.machines.machinesCyan,
+      currPhase: state => state.phase,
+      ringspecs: state => state.machines.ringspecs
+    })
+  },
+
+  data() {
+    return {
+      imageUrl: 'http://via.placeholder.com/50x25'
+    }
+  },
+  // Run on mount of Component
+  mounted() {
+    this.fetchMachinesCyan()
+    this.fetchRingSpec()
+    this.pollMachineInfo();
+  },
+
+  methods: {
+    ...mapActions({
+      fetchMachinesCyan: 'fetchMachinesCyan',
+      fetchRingSpec: 'fetchRingSpec',
+    }),
+    pollMachineInfo() {
+      // Polls Information every 1,5 seconds
+      setInterval(this.fetchMachinesCyan, 1500)
```

Comment:
maybe you can add a variable for the polling wait time, I think that might be a bit more clean.